src/binding/c/rma_api.txt

```diff
@@ -0,0 +1,588 @@
+    .seealso: MPI_Put
+    .impl: mpid
+    .earlyreturn: pt2pt_proc_null
+
+MPI_Win_allocate:
+    .desc: Create and allocate an MPI Window object for one-sided communication
+    .seealso: MPI_Win_allocate_shared MPI_Win_create MPI_Win_create_dynamic MPI_Win_free
+    .impl: mpid
+
+MPI_Win_allocate_shared:
+    .desc: Create an MPI Window object for one-sided communication and shared memory access, and allocate memory at each process
+    .seealso: MPI_Win_allocate MPI_Win_create MPI_Win_create_dynamic MPI_Win_free MPI_Win_shared_query
+    .impl: mpid
+/*
+    This is a collective call executed by all processes in the group of comm. On
+    each process i, it allocates memory of at least size bytes that is shared among
+    all processes in comm, and returns a pointer to the locally allocated segment
+    in baseptr that can be used for load/store accesses on the calling process. The
+    locally allocated memory can be the target of load/store accesses by remote
+    processes; the base pointers for other processes can be queried using the
+    function 'MPI_Win_shared_query'.
+
+    The call also returns a window object that can be used by all processes in comm
+    to perform RMA operations. The size argument may be different at each process
+    and size = 0 is valid. It is the user''s responsibility to ensure that the
+    communicator comm represents a group of processes that can create a shared
+    memory segment that can be accessed by all processes in the group. The
+    allocated memory is contiguous across process ranks unless the info key
+    alloc_shared_noncontig is specified. Contiguous across process ranks means that
+    the first address in the memory segment of process i is consecutive with the
+    last address in the memory segment of process i - 1. This may enable the user
+    to calculate remote address offsets with local information only.
+*/
+
+MPI_Win_attach:
+    .desc: Attach memory to a dynamic window
+    .seealso: MPI_Win_create_dynamic MPI_Win_detach
+    .impl: mpid
+{ -- early_return --
+    if (size == 0)
+        goto fn_exit;
+}
+
+MPI_Win_complete:
+    .desc: Completes an RMA operations begun after an MPI_Win_start
+    .impl: mpid
+
+MPI_Win_create:
+    .desc: Create an MPI Window object for one-sided communication
+    .seealso: MPI_Win_allocate MPI_Win_allocate_shared MPI_Win_create_dynamic MPI_Win_free
+    .impl: mpid
+/*
+    Notes:
+    
+    The displacement unit argument is provided to facilitate address arithmetic in
+    RMA operations: the target displacement argument of an RMA operation is scaled
+    by the factor disp_unit specified by the target process, at window creation.
+    
+    The info argument provides optimization hints to the runtime about the expected
+    usage pattern of the window. The following info keys are predefined.
+    
+    . no_locks - If set to true, then the implementation may assume that passive
```

Comment:
I generated the files again locally. Except a few issues that we just discussed early today, I think the PR is ready to go. One minor comment is that there are lines with only whitespace in the `Notes` area of `src/binding/c/rma_api.txt`. Please cleanup them.

Reply:
Fixed.